src/shapes/polyline.class.ts

```diff
@@ -142,6 +142,16 @@
       };
     },
 
+    /**
+     * After setting scale, recalculates dimensions when the stroke is uniform and stroke line join is bevel or round
+     * @private
+     */
+    _set(key, value) {
+      var output = this.callSuper('_set', key, value);
+      (key === 'scaleX' || key === 'scaleY') && this.strokeUniform && this.strokeLineJoin !== 'round' && this._setPositionDimensions();
```

Comment:
does `skew` affect as well?

Reply:
Seems we didn't take that into account.
Want to focus on it in a follow up or here?

Reply:
skew bbox is inaccurate regardless I think